packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -761,6 +763,47 @@ describe('plugin-meetings', () => {
 
         assert(loggerSpy.neverCalledWith('StatsAnalyzer:index#compareLastStatsResult --> No audio RTP packets received'));
       });
+
+      it('logs a message when no packets are recieved for a receive slot with sourceState "live"', async () => {
+        receiveSlot = {
+          sourceState:  'live',
+          csi: 2,
+          id: "4",
+        };
+
+        await startStatsAnalyzer();
+
+        // don't increase the packets when time progresses.
+        await progressTime();
+
+        assert.calledWith(loggerSpy, 'StatsAnalyzer:index#processInboundRTPResult --> No packets received for receive slot id: "4" and csi: 2');
```

Comment:
shouldn't there be 1 more parameter passed to loggerSpy? (the `currentPacketsReceived` that I made a comment about above)

Reply:
Yes, I've added it for the tests I've added now